CHANGES.rst

```diff
@@ -1,14 +1,128 @@
+Different units for the energy axis of the counts and exposure map in a `MapDataset`
+are now handled correctly. We also fixed the equations of the pulsar spin-down
+model in `gammapy.astro.source`.
+
+We unified the handling of sky regions in Gammapy, as described in `PIG 10`_.
+For convenience regions can now also be created from DS9 region strings. We
+continued the clean-up process of `gammapy.spectrum` and removed the
+`PHACountsSpectrum` class, which is now fully replaced by the `SpectrumDatasetOnOff`
+class. We also removed the `Energy` and `EnergyBounds` classes. Grids of energies
+can be created and handled directly now using the `MapAxis` object.
+
+We added a new `SpectralGaussian` and `PLSuperExpCutoff4FG` model and changed
+the definition of the position angle of the `SkyEllipse` model to follow IAU
+conventions. To support binned simulation of counts data in a uniform way we
+implemented `MapDataset.fake()`, `SpectrumDataset.fake()` and `SpectrumDatasetOnOff.fake()`
+methods, which simulate binned counts maps and spectra from models. In addition
+we implemented nice string representations for all of our dataset classes and
+added convenience functions to compute residuals using different methods on all
+of them.
+
+We updated the recommended conda environment for Gammapy v0.13. We now rely
+on Python 3.7, Ipython 7.5, Scipy 1.3, Matplotlib 3.1, Astropy 3.1, and Healpy 1.12.
+These updates should be backwards compatible, your scripts and notebooks should
+run and give the same results.
+
+We also actively continued to write PIG documents to finish the development and
+feature planning for Gammapy v1.0. This includes proposals for a high level
+interface, improvements in data reduction, event sampling and time analysis.
+We already started the implementation of YAML serialisation for models,
+event sampling and lightcurve estimation.
+
+The Gammapy developer team will go to a well earned summer break now, but after
+the break we will continue developing fast. So stay tuned for the next Gammapy
+release in September!
+
+
+**Contributors:**
+
+In alphabetical order by first name:
+
+- Atreyee Sinha
+- Axel Donath
+- Brigitta Sipocz
+- Bruno Khelifi
+- Christoph Deil
+- Fabio Pintore
+- Fabio Acero
+- Kaori Nakashima
+- José Enrique Ruiz
+- Léa Jouvin
+- Luca Giunti
+- Quentin Remy
+- Régis Terrier
+- Silvia Manconi
+- Yu Wun Wong
+
+
+
+Pull Requests
++++++++++++++
+
+This list is incomplete. Small improvements and bug fixes are not listed here.
 
-The recommended conda environment for Gammapy v0.13 contains some version updates:
-Python 3.7, ipython 7.5, Scipy 1.3, Matplotlib 3.1, Astropy 3.1, healpy 1.12.
-These updates should be backward compatible, your scripts and notebooks should
-run and give the same outputs.
+See the complete `Gammapy v0.13 merged pull requests list on Github https://github.com/gammapy/gammapy/pulls?utf8=✓&q=is%3Apr+milestone%3A0.13+`__.
+
+
+- [#2310] Remove old `LightCurveEstimator` class (Axel Donath)
```

Comment:
If you can wait an hour, please list https://github.com/gammapy/gammapy/pull/2296, I'd like to merge it in today.

Reply:
Yes, we will wait. I added it to the changelog...